ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-isis-new.cfg

```diff
@@ -79,7 +79,7 @@ router isis EVPN_UNDERLAY
    timers local-convergence-delay 15000 protected-prefixes
    set-overload-bit on-startup wait-for-bgp timeout 10
    advertise passive-only
-   spf-interval 250 10
+   spf-interval 250 10 20
```

Comment:
Why do we  have another `router-isis` file?

Reply:
Not sure why we have two files.